### PR TITLE
Add close button on tradeTabs

### PR DESCRIPTION
**Pull request by DanSallau**
Hi ,

Attached contained few changes to address the issue here https://trello.com/c/pU3PzMDA/37-cannot-close-multiple-trade-windows-for-different-assets as reported by the QA. Should there be any issue with regard to this, please do let me know and would be glad to address.

### CODE REVIEW

File: src/_common/Tab.js [hunk 1]
```diff
@@ -28,6 +32,11 @@ export default class Tab extends Component {
             >
                 {showIcon && imgSrc && <img src={imgSrc} />}
                 {showText && text && <span>{text}</span>}
+                {showCloseIcon &&   
+                    (<button className="close-btn" onClick={onClose}>
```
Comment (borisyankov) [inline src/_common/Tab.js]: You don't need to put () around button

File: src/_common/__tests__/Tab-test.js [hunk 2]
```diff
@@ -19,6 +19,16 @@ describe('<Tab />', () => {
         expect(wrapper.text()).to.equal('');
     });
 
+    it('should render tab with close button', () => {
```
Comment (borisyankov) [inline src/_common/__tests__/Tab-test.js]: Tests. Good.
Be more descriptive. This is almost like documentation.
Good text here allows us to not write comments in the actual source code.
In that case I might call it 'renders a close button when showCloseIcon is true'.

File: src/trades/TradesTabs.js [hunk 3]
```diff
@@ -28,6 +28,11 @@ export default class TradesTabs extends Component {
                         <Tab
                             key={index}
                             text={trade.symbolName}
+                            showCloseIcon
```
Comment (borisyankov) [inline src/trades/TradesTabs.js]: Rename to 'closable' everywhere :)

File: src/trades/TradesTabs.js [hunk 4]
```diff
@@ -28,6 +28,11 @@ export default class TradesTabs extends Component {
...
+                            onClose={ev => {
+                                actions.removeTrade(index);
+                                ev.stopPropagation();
```
Comment (borisyankov) [inline src/trades/TradesTabs.js]: What happens if you don't call stopPropagation. Is it necessary?

Comment (nuruddeensalihu) [inline src/trades/TradesTabs.js]: Used to adding those in js buttons to address event bubbling issue [https://developer.mozilla.org/en/docs/Web/API/Event/stopPropagation] here. Didn't try without it in react though . :wink:

**Pull request merged**